src/Nest/XPack/MachineLearning/Job/Results/BucketInfluencer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	[JsonObject]
+	public class BucketInfluencer
+	{
+		[JsonProperty("job_id")]
```

Comment:
This PR is full of awesome xmldocs but this one is left out 😿

Reply:
Done.